yb-voyager/src/srcdb/ora2pg_extract_schema.go

```diff
@@ -13,8 +15,15 @@ import (
 
 func ora2pgExtractSchema(source *Source, exportDir string) {
 	schemaDirPath := exportDir + "/schema"
+	customConfigFilePath := exportDir + "/metainfo/base_ora2pg.conf"
```

Comment:
prefer using `filepath.Join()` as a best practice and no need to worry about `/`